src/pages/lock-exceptions/LockExceptions.js

```diff
@@ -0,0 +1,137 @@
+import { useDataQuery } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import {
+    CenteredContent,
+    CircularLoader,
+    NoticeBox,
+    Button,
+    Pagination,
+    ButtonStrip,
+} from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React, { useEffect, useState } from 'react'
+import { Link } from 'react-router-dom'
+import LockExceptionsTable from '../../components/LockExceptionsTable/LockExceptionsTable'
+import PageHeader from '../../components/PageHeader/PageHeader'
+import RemoveLockExceptionModal from '../../components/RemoveLockExceptionModal/RemoveLockExceptionModal'
+import { i18nKeys } from '../../i18n-keys'
+import { parseLockExceptions } from '../../parse-lock-exceptions'
+import styles from './LockExceptions.module.css'
+
+const query = {
+    lockExceptions: {
+        resource: 'lockExceptions',
+        params: params => ({
+            ...params,
+            fields:
+                'name,period[id,displayName],organisationUnit[id,displayName],dataSet[id,displayName]',
+            order: 'name:asc',
+        }),
+    },
+}
+
+const LockExceptions = ({ sectionKey }) => {
```

Comment:
This component is a rewrite of https://github.com/dhis2/data-administration-app/blob/7a6f8169e83609ea241f83da4ac2be96ac337fbf/src/pages/lock-exception/LockException.js